demos/node-classification/rgcn/rgcn-aifb-node-classification-example.ipynb

```diff
@@ -246,56 +246,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": null,
```

Comment:
This hasn't removed the warning in the text at the start of the file. In fact, I don't think any of this needs to change because the code isn't changing?

```
git checkout HEAD^ -- demos/node-classification/rgcn/rgcn-aifb-node-classification-example.ipynb
```

will get the file back to what it was before rerunning.

Reply:
Ah I misread your last review, I thought you were referring to code warnings appearing in the demo notebook. I re-ran notebook to be sure there were no experimental and so I could keep more output visible (less warnings so less cells I had to clear).

I'll delete that warning text at the start

Reply:
Ah, makes sense 👍